src/main/java/frc/trigon/robot/subsystems/shooter/ShooterConstants.java

```diff
@@ -0,0 +1,42 @@
+package frc.trigon.robot.subsystems.shooter;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+public class ShooterConstants {
+
+    private static final int
+            LEFT_MOTOR_ID = 3,
+            RIGHT_MOTOR_ID = 4;
+
+    private static final double VOLTAGE_COMPENSATION = 0;
+
+    private static final double
+            P = 0,
+            I = 0,
+            D = 0,
+            V = 0;
+
+    static final double S = 0;
+
+    static final WPI_TalonFX
+            LEFT_MOTOR = new WPI_TalonFX(LEFT_MOTOR_ID),
+            RIGHT_MOTOR = new WPI_TalonFX(RIGHT_MOTOR_ID),
+            MASTER_MOTOR = RIGHT_MOTOR,
```

Comment:
separate master and follower from left and right.
master is the only one that should not be private.